lispify/lispify.py

```diff
@@ -9,11 +9,18 @@
 so that the lispify method ignores it.
 """
 
+from __future__ import absolute_import
+
 from numbers import Number
+from sys import version_info
 import warnings
 
-from util import subclasses, camel_case_to_lisp_name
+from lispify.util import subclasses, camel_case_to_lisp_name
+
 
+# python 3 compatibility functions
+def string_types():
```

Comment:
See http://python-future.org/faq.html#what-is-the-relationship-between-future-and-six. Although both strive towards the same goal, the playing around I did leads me to agree with `future`'s claim that they have a (better) higher level API with more features. 

I'm totally okay with 3rd party libraries, as the advantages I listed above seem to outweigh the one disadvantage.